public/about.html

```diff
@@ -0,0 +1,47 @@
+<html>
+<head>
+  <title>About</title>
+  <!-- <meta charset="utf-8">
+  <meta name="description" content="About Loraine Kanervisto, a writer and future software developer living in Seattle.">
+  <meta name="keywords" content="programming, ruby, writing, bicycles">
+  <meta name="author" content="Loraine Kanervisto"> -->
+</head>
+
+  <body>
+
+    <header>
+      <h1>About Me</h1>
+      <nav>
+        <ul>
+          <li><a href="/">Home</a></li>
+          <li><a href="programming">Programming</a></li>
+          <li><a href="writing">Writing</a></li>
+        </ul>
+      </nav>
+    </header>
+
+  <section> <!-- About me blurb -->
+    <header>
+      <h2>Parts of Speech</h2>
+    </header>
+
+    <table>
+      <tr><td>Verbs</td><td>Pronouns</td><td>Nouns</td><td>Adjectives</td><td>Prepositional Phrases</td><tr>
```

Comment:
I dunno why I did that. Are we going to be able to incorporate markdown into our projects later?